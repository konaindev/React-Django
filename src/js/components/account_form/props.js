export const props = {
  steps: [
    { name: "Set Password", isComplete: true },
    { name: "Complete Account", isActive: true }
  ],
  title: "Complete your account",
  subtitle:
    "We need just a bit more information about you to complete your account."
};
