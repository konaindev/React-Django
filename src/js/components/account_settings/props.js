const user = {
  account_id: 1,
  account_name: "Remarkably",
  account_settings_url: "/users/account-settings",
  email: "test@psl.com",
  logout_url: "/users/logout/",
  user_id: "usr_jjzpeyfeshzpaha5"
};

const itemsOrder = ["profile", "lock", "email"];

const rules = [
  {
    label: "Be at least 8 characters",
    key: "length"
  },
  {
    label: "Contain alphabetic characters",
    key: "characters"
  },
  {
    label: "Not match personal information",
    key: "personal"
  },
  {
    label: "Not be a commonly used password",
    key: "used"
  }
];

const company_roles = [
  { label: "Owner", value: "owner" },
  { label: "Developer", value: "developer" },
  { label: "Asset Manager", value: "asset_manager" },
  { label: "Property Manager", value: "property_manager" }
];

const office_options = [
  { label: "Global", value: 1 },
  { label: "National", value: 2 },
  { label: "Regional", value: 3 },
  { label: "Other", value: 4 }
];

const office_countries = [
  { label: "United States of America", value: "USA" },
  { label: "United Kingdom", value: "GBR" }
];

const us_state_list = [
  { label: "Washington", value: "Washington" },
  { label: "New York", value: "New York" }
];

const gb_county_list = [
  { label: "London", value: "London" },
  { label: "Aberdeen", value: "Aberdeen" }
];

const profile = {
  avatar_url:
    "https://lh3.googleusercontent.com/-cQLcFi7r2uc/AAAAAAAAAAI/AAAAAAAAAAA/ACHi3rfoCSVbR8qVruV55uAYdSC-znVn2w.CMID/s96-c/photo.jpg",
  first_name: "Phillip",
  last_name: "McPhillipson",
  title: "Founder",
  phone_country_code: "",
  phone: "",
  phone_ext: "",
  company: { label: "Glacier Associates", value: "123456789" },
  company_roles: ["owner", "asset_manager", "property_manager"],
  office_street: "1730 Minor Avenue",
  office_city: "Lansing",
  office_state: { label: "Michigan", value: "Michigan" },
  office_zip: "11111",
  office_country: { label: "United States of America", value: "USA" },
  office_address: "1730 Minor Avenue, Lansing, MI",
  office_name: "Michigan",
  office_type: 3
};

const tabsOrder = ["portfolio", "group", "property"];

const initialTab = "portfolio";

export const props = {
  rules,
  profile,
  user,
  company_roles,
  office_options,
  itemsOrder,
  tabsOrder,
  initialTab,
  office_countries,
  us_state_list,
  gb_county_list,
  dispatch: () => {}
};
