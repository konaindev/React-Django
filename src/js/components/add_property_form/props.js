export const props = {
  post_url: "/projects/new",
  states: [
    { id: "AL", name: "AL" },
    { id: "GA", name: "GA" },
    { id: "NE", name: "NE" },
    { id: "OR", name: "OR" },
    { id: "WY", name: "WY" }
  ],
  packages: [
    { id: "accelerate", name: "Accelerate" },
    { id: "optimize", name: "Optimize" },
    { id: "ground", name: "Ground Up" },
    { id: "other", name: "Not Sure" }
  ]
};
