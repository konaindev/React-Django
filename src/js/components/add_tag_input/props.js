const suggestedTags = [
  { name: "Tag 1", count: 5 },
  { name: "Test Tag 2", count: 3 },
  { name: "Lake Highland Properties 3", count: 2 },
  { name: "New Tag", count: 1 }
];

export default { suggestedTags };
