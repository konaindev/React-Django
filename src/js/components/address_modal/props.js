export default {
  isOpen: true,
  addresses: {
    suggested_address: {
      office_city: "Washington",
      office_state: "DC",
      office_street: "1600 Pennsylvania Ave NW",
      office_zip: "20500",
      office_country: "United States"
    }
  },
  dispatch_type: "FAKE DISPATCH"
};
