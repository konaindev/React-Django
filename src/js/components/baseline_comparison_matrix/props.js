import {
  baselineReport,
  baselineReportWithoutCompetitors,
  baselineReportWithOneCompetitor
} from "../common_report/props";

export const props = { report: baselineReport };
export const no_competitor_props = { report: baselineReportWithoutCompetitors };
export const one_competitor_props = {
  report: baselineReportWithOneCompetitor
};
