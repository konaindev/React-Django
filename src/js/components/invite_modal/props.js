const members = [
  {
    user_id: "peep_00001",
    profile_image_url: "/photo.jpg",
    account_name: "Remarkably Client",
    email: "client@remarkably.com",
    role: "member"
  },
  {
    user_id: "peep_00002",
    account_name: "Remarkably Client",
    email: "client@remarkably.com",
    role: "member"
  },
  {
    user_id: "peep_00003",
    profile_image_url:
      "https://lh3.googleusercontent.com/-cQLcFi7r2uc/AAAAAAAAAAI/AAAAAAAAAAA/ACHi3rfoCSVbR8qVruV55uAYdSC-znVn2w.CMID/s96-c/photo.jpg",
    account_name: "Remarkably Client",
    email: "client@remarkably.com",
    role: "admin"
  },
  {
    user_id: "peep_00004",
    profile_image_url:
      "https://lh3.googleusercontent.com/-cQLcFi7r2uc/AAAAAAAAAAI/AAAAAAAAAAA/ACHi3rfoCSVbR8qVruV55uAYdSC-znVn2w.CMID/s96-c/photo.jpg",
    account_name: "Remarkably Client",
    email: "client@remarkably.com",
    role: "member"
  },
  {
    user_id: "peep_00005",
    profile_image_url:
      "https://lh3.googleusercontent.com/-cQLcFi7r2uc/AAAAAAAAAAI/AAAAAAAAAAA/ACHi3rfoCSVbR8qVruV55uAYdSC-znVn2w.CMID/s96-c/photo.jpg",
    account_name: "Remarkably Client",
    email: "client@remarkably.com",
    role: "member"
  },
  {
    user_id: "peep_00006",
    profile_image_url:
      "https://lh3.googleusercontent.com/-cQLcFi7r2uc/AAAAAAAAAAI/AAAAAAAAAAA/ACHi3rfoCSVbR8qVruV55uAYdSC-znVn2w.CMID/s96-c/photo.jpg",
    account_name: "Remarkably Client",
    email: "client@remarkably.com",
    role: "member"
  },
  {
    user_id: "peep_00007",
    profile_image_url:
      "https://lh3.googleusercontent.com/-cQLcFi7r2uc/AAAAAAAAAAI/AAAAAAAAAAA/ACHi3rfoCSVbR8qVruV55uAYdSC-znVn2w.CMID/s96-c/photo.jpg",
    account_name: "Remarkably Client",
    email: "client@remarkably.com",
    role: "member"
  },
  {
    user_id: "peep_00008",
    profile_image_url:
      "https://lh3.googleusercontent.com/-cQLcFi7r2uc/AAAAAAAAAAI/AAAAAAAAAAA/ACHi3rfoCSVbR8qVruV55uAYdSC-znVn2w.CMID/s96-c/photo.jpg",
    account_name: "Remarkably Client",
    email: "client@remarkably.com",
    role: "member"
  },
  {
    user_id: "peep_00009",
    profile_image_url:
      "https://lh3.googleusercontent.com/-cQLcFi7r2uc/AAAAAAAAAAI/AAAAAAAAAAA/ACHi3rfoCSVbR8qVruV55uAYdSC-znVn2w.CMID/s96-c/photo.jpg",
    account_name: "Remarkably Client",
    email: "client@remarkably.com",
    role: "member"
  },
  {
    user_id: "peep_00010",
    profile_image_url:
      "https://lh3.googleusercontent.com/-cQLcFi7r2uc/AAAAAAAAAAI/AAAAAAAAAAA/ACHi3rfoCSVbR8qVruV55uAYdSC-znVn2w.CMID/s96-c/photo.jpg",
    account_name: "Remarkably Client",
    email: "client@remarkably.com",
    role: "member"
  },
  {
    user_id: "peep_00011",
    profile_image_url:
      "https://lh3.googleusercontent.com/-cQLcFi7r2uc/AAAAAAAAAAI/AAAAAAAAAAA/ACHi3rfoCSVbR8qVruV55uAYdSC-znVn2w.CMID/s96-c/photo.jpg",
    account_name: "Remarkably Client",
    email: "client@remarkably.com",
    role: "member"
  },
  {
    user_id: "peep_00012",
    profile_image_url:
      "https://lh3.googleusercontent.com/-cQLcFi7r2uc/AAAAAAAAAAI/AAAAAAAAAAA/ACHi3rfoCSVbR8qVruV55uAYdSC-znVn2w.CMID/s96-c/photo.jpg",
    account_name: "Remarkably Client",
    email: "client@remarkably.com",
    role: "member"
  },
  {
    user_id: "peep_00013",
    profile_image_url:
      "https://lh3.googleusercontent.com/-cQLcFi7r2uc/AAAAAAAAAAI/AAAAAAAAAAA/ACHi3rfoCSVbR8qVruV55uAYdSC-znVn2w.CMID/s96-c/photo.jpg",
    account_name: "Remarkably Client",
    email: "client@remarkably.com",
    role: "member"
  }
];

export const props = {
  open: true,
  properties: [
    {
      property_name: "The Chalet",
      members: members
    }
  ]
};

export const multiProps = {
  open: true,
  properties: [
    {
      property_name: "Del Mar",
      members: members.slice(0, 1)
    },
    {
      property_name: "The Chalet",
      members: members.slice(1)
    }
  ]
};
