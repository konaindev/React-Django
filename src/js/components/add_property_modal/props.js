import { states } from "../add_property_form/states";

export const props = {
  formProps: {
    post_url: null,
    packages: [
      { id: "accelerate", name: "Accelerate" },
      { id: "optimize", name: "Optimize" },
      { id: "ground", name: "Ground Up" },
      { id: "other", name: "Not Sure" }
    ],
    states
  }
};
