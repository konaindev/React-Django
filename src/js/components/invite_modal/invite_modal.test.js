import React from "react";
import renderer from "react-test-renderer";
import { createStore } from "redux";
import { Provider } from "react-redux";

import InviteModal from "./index";
import { props, multiProps } from "./props";

jest.mock("react-responsive-modal", () => "Modal");

const _ = x => createStore(() => x);

describe("InviteModal", () => {
  it("renders with single properties", () => {
    const tree = renderer
      .create(
        <Provider store={_(props)}>
          <InviteModal />
        </Provider>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
  it("renders with multiple properties", () => {
    const tree = renderer
      .create(
        <Provider store={_(multiProps)}>
          <InviteModal />
        </Provider>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
