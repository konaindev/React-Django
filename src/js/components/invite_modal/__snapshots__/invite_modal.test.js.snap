// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InviteModal renders with multiple properties 1`] = `
Array [
  <Modal
    center={true}
    classNames={
      Object {
        "closeButton": "modal-window__close",
        "modal": "modal-window modal-window--small invite-modal",
        "overlay": "modal-overlay",
      }
    }
    closeOnOverlayClick={true}
    focusTrapped={false}
    onClose={[Function]}
    onEntered={[Function]}
    open={true}
    styles={
      Object {
        "closeIcon": Object {
          "display": "none",
        },
      }
    }
  >
    <div
      className="modal-window__head invite-modal__header"
    >
      <div
        className="invite-modal__title"
      >
        Invite to
      </div>
      <div
        className="invite-modal__title invite-modal__title--name"
      >
         
        0 properties
      </div>
    </div>
    <div
      className="modal-window__body"
    >
      <div
        className="invite-modal__container invite-modal__container--select"
      >
        <div
          className="select-search select invite-modal__select-users select--is-multi select--transparent css-2b097c-container"
          onKeyDown={[Function]}
        >
          <div
            className="select__control css-yk16xz-control"
            onMouseDown={[Function]}
            onTouchEnd={[Function]}
          >
            <div
              className="select__value-container select__value-container--is-multi css-1hwfws3"
            >
              <div
                className="select__input-with-placeholder"
              >
                <div
                  className="css-zz0hea-Input"
                >
                  <div
                    className="select__input"
                    style={
                      Object {
                        "display": "inline-block",
                      }
                    }
                  >
                    <input
                      aria-autocomplete="list"
                      autoCapitalize="none"
                      autoComplete="off"
                      autoCorrect="off"
                      disabled={false}
                      id="react-select-4-input"
                      onBlur={[Function]}
                      onChange={[Function]}
                      onFocus={[Function]}
                      spellCheck="false"
                      style={
                        Object {
                          "background": 0,
                          "border": 0,
                          "boxSizing": "content-box",
                          "color": "inherit",
                          "fontSize": "inherit",
                          "label": "input",
                          "opacity": 1,
                          "outline": 0,
                          "padding": 0,
                          "width": "1px",
                        }
                      }
                      tabIndex="0"
                      type="text"
                      value=""
                    />
                    <div
                      style={
                        Object {
                          "height": 0,
                          "left": 0,
                          "overflow": "scroll",
                          "position": "absolute",
                          "top": 0,
                          "visibility": "hidden",
                          "whiteSpace": "pre",
                        }
                      }
                    >
                      
                    </div>
                  </div>
                </div>
                <div
                  className="select__placeholder css-1wa3eu0-placeholder"
                >
                  Name or email
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          className="select invite-modal__select-role select--small select--default css-2b097c-container"
          onKeyDown={[Function]}
        >
          <div
            className="select__control css-yk16xz-control"
            onMouseDown={[Function]}
            onTouchEnd={[Function]}
          >
            <div
              className="select__value-container select__value-container--has-value css-1hwfws3"
            >
              <div
                className="select__single-value css-1uccc91-singleValue"
              >
                Member
              </div>
              <input
                className="css-62g3xt-dummyInput"
                disabled={false}
                id="react-select-5-input"
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                readOnly={true}
                tabIndex="0"
                value=""
              />
            </div>
            <div
              className="select__indicators css-1g48xl4-IndicatorsContainer"
            >
              <span
                className="select__indicator-separator css-1okebmr-indicatorSeparator"
              />
              <div
                aria-hidden="true"
                className="select__indicator select__dropdown-indicator css-tlfecz-indicatorContainer"
                onMouseDown={[Function]}
                onTouchEnd={[Function]}
              >
                <svg
                  aria-hidden="true"
                  className="select__dropdown-arrow select__dropdown-arrow--down"
                  color="inherit"
                  focusable="false"
                  height="1em"
                  role="presentation"
                  viewBox="0 0 10 5"
                  width="1em"
                >
                  <path
                    d="M1 1.16134L4.92334 4.5"
                    stroke="currentColor"
                    strokeLinecap="round"
                  />
                  <path
                    d="M8.82812 1.16134L5 4.5"
                    stroke="currentColor"
                    strokeLinecap="round"
                  />
                </svg>
                <svg
                  aria-hidden="true"
                  className="select__dropdown-arrow select__dropdown-arrow--up"
                  color="inherit"
                  focusable="false"
                  height="1em"
                  role="presentation"
                  viewBox="0 0 10 5"
                  width="1em"
                >
                  <path
                    d="M1 3.83866L4.92334 0.5"
                    stroke="currentColor"
                    strokeLinecap="round"
                  />
                  <path
                    d="M8.82812 3.83866L5 0.5"
                    stroke="currentColor"
                    strokeLinecap="round"
                  />
                </svg>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        className="invite-modal__container invite-modal__container--users"
      />
      <div
        className="invite-modal__container invite-modal__container--button"
      >
        <button
          className="button button--disabled button--uppercase button--primary invite-modal__button"
          onClick={[Function]}
        >
          invite
        </button>
      </div>
    </div>
  </Modal>,
  <Modal
    center={true}
    classNames={
      Object {
        "closeButton": "modal-window__close",
        "modal": "modal-window modal-window--small invite-remove-window",
        "overlay": "modal-overlay",
      }
    }
    closeOnOverlayClick={true}
    focusTrapped={false}
    onClose={[Function]}
    open={false}
    styles={
      Object {
        "closeIcon": Object {
          "display": "none",
        },
      }
    }
  >
    <div
      className="modal-window__head"
    >
      Are you sure?
    </div>
    <div
      className="modal-window__body invite-remove-window__body"
    >
      Removing
       
      <span
        className="invite-remove-window__name"
      />
       
      will revoke their access to this property.
      <div
        className="invite-remove-window__controls"
      >
        <button
          className="button button--uppercase button--secondary invite-remove-window__button"
          onClick={[Function]}
        >
          Cancel
        </button>
        <button
          className="button button--uppercase button--warning invite-remove-window__button"
          onClick={[Function]}
        >
          Remove
        </button>
      </div>
    </div>
  </Modal>,
]
`;

exports[`InviteModal renders with single properties 1`] = `
Array [
  <Modal
    center={true}
    classNames={
      Object {
        "closeButton": "modal-window__close",
        "modal": "modal-window modal-window--small invite-modal",
        "overlay": "modal-overlay",
      }
    }
    closeOnOverlayClick={true}
    focusTrapped={false}
    onClose={[Function]}
    onEntered={[Function]}
    open={true}
    styles={
      Object {
        "closeIcon": Object {
          "display": "none",
        },
      }
    }
  >
    <div
      className="modal-window__head invite-modal__header"
    >
      <div
        className="invite-modal__title"
      >
        Invite to
      </div>
      <div
        className="invite-modal__title invite-modal__title--name"
      >
         
        0 properties
      </div>
    </div>
    <div
      className="modal-window__body"
    >
      <div
        className="invite-modal__container invite-modal__container--select"
      >
        <div
          className="select-search select invite-modal__select-users select--is-multi select--transparent css-2b097c-container"
          onKeyDown={[Function]}
        >
          <div
            className="select__control css-yk16xz-control"
            onMouseDown={[Function]}
            onTouchEnd={[Function]}
          >
            <div
              className="select__value-container select__value-container--is-multi css-1hwfws3"
            >
              <div
                className="select__input-with-placeholder"
              >
                <div
                  className="css-zz0hea-Input"
                >
                  <div
                    className="select__input"
                    style={
                      Object {
                        "display": "inline-block",
                      }
                    }
                  >
                    <input
                      aria-autocomplete="list"
                      autoCapitalize="none"
                      autoComplete="off"
                      autoCorrect="off"
                      disabled={false}
                      id="react-select-2-input"
                      onBlur={[Function]}
                      onChange={[Function]}
                      onFocus={[Function]}
                      spellCheck="false"
                      style={
                        Object {
                          "background": 0,
                          "border": 0,
                          "boxSizing": "content-box",
                          "color": "inherit",
                          "fontSize": "inherit",
                          "label": "input",
                          "opacity": 1,
                          "outline": 0,
                          "padding": 0,
                          "width": "1px",
                        }
                      }
                      tabIndex="0"
                      type="text"
                      value=""
                    />
                    <div
                      style={
                        Object {
                          "height": 0,
                          "left": 0,
                          "overflow": "scroll",
                          "position": "absolute",
                          "top": 0,
                          "visibility": "hidden",
                          "whiteSpace": "pre",
                        }
                      }
                    >
                      
                    </div>
                  </div>
                </div>
                <div
                  className="select__placeholder css-1wa3eu0-placeholder"
                >
                  Name or email
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          className="select invite-modal__select-role select--small select--default css-2b097c-container"
          onKeyDown={[Function]}
        >
          <div
            className="select__control css-yk16xz-control"
            onMouseDown={[Function]}
            onTouchEnd={[Function]}
          >
            <div
              className="select__value-container select__value-container--has-value css-1hwfws3"
            >
              <div
                className="select__single-value css-1uccc91-singleValue"
              >
                Member
              </div>
              <input
                className="css-62g3xt-dummyInput"
                disabled={false}
                id="react-select-3-input"
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                readOnly={true}
                tabIndex="0"
                value=""
              />
            </div>
            <div
              className="select__indicators css-1g48xl4-IndicatorsContainer"
            >
              <span
                className="select__indicator-separator css-1okebmr-indicatorSeparator"
              />
              <div
                aria-hidden="true"
                className="select__indicator select__dropdown-indicator css-tlfecz-indicatorContainer"
                onMouseDown={[Function]}
                onTouchEnd={[Function]}
              >
                <svg
                  aria-hidden="true"
                  className="select__dropdown-arrow select__dropdown-arrow--down"
                  color="inherit"
                  focusable="false"
                  height="1em"
                  role="presentation"
                  viewBox="0 0 10 5"
                  width="1em"
                >
                  <path
                    d="M1 1.16134L4.92334 4.5"
                    stroke="currentColor"
                    strokeLinecap="round"
                  />
                  <path
                    d="M8.82812 1.16134L5 4.5"
                    stroke="currentColor"
                    strokeLinecap="round"
                  />
                </svg>
                <svg
                  aria-hidden="true"
                  className="select__dropdown-arrow select__dropdown-arrow--up"
                  color="inherit"
                  focusable="false"
                  height="1em"
                  role="presentation"
                  viewBox="0 0 10 5"
                  width="1em"
                >
                  <path
                    d="M1 3.83866L4.92334 0.5"
                    stroke="currentColor"
                    strokeLinecap="round"
                  />
                  <path
                    d="M8.82812 3.83866L5 0.5"
                    stroke="currentColor"
                    strokeLinecap="round"
                  />
                </svg>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        className="invite-modal__container invite-modal__container--users"
      />
      <div
        className="invite-modal__container invite-modal__container--button"
      >
        <button
          className="button button--disabled button--uppercase button--primary invite-modal__button"
          onClick={[Function]}
        >
          invite
        </button>
      </div>
    </div>
  </Modal>,
  <Modal
    center={true}
    classNames={
      Object {
        "closeButton": "modal-window__close",
        "modal": "modal-window modal-window--small invite-remove-window",
        "overlay": "modal-overlay",
      }
    }
    closeOnOverlayClick={true}
    focusTrapped={false}
    onClose={[Function]}
    open={false}
    styles={
      Object {
        "closeIcon": Object {
          "display": "none",
        },
      }
    }
  >
    <div
      className="modal-window__head"
    >
      Are you sure?
    </div>
    <div
      className="modal-window__body invite-remove-window__body"
    >
      Removing
       
      <span
        className="invite-remove-window__name"
      />
       
      will revoke their access to this property.
      <div
        className="invite-remove-window__controls"
      >
        <button
          className="button button--uppercase button--secondary invite-remove-window__button"
          onClick={[Function]}
        >
          Cancel
        </button>
        <button
          className="button button--uppercase button--warning invite-remove-window__button"
          onClick={[Function]}
        >
          Remove
        </button>
      </div>
    </div>
  </Modal>,
]
`;
